packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -376,9 +375,10 @@ export default function (
               run(task);
             }
 
-            defaults(task.options, {
+            task.options = {
               sourceMap: !!(task.sourceMappings || task.sourceMap),
-            });
+              ...task.options,
+            };
```

Comment:
This introduces some slight changes to the behavior though I'm not sure it matters in this module / context.  Prior to this PR `task.options` got mutated, this PR causes it to instead be cloned and `task` gets mutated.  I think the exact replacement for existing code would actually be:
```js
if ('sourceMap' in task.options === false) {
  task.options.sourceMap = !!(task.sourceMappings || task.sourceMap);
}
```

I'm not familiar with this module to say if this is actually an issue so I'm not actually requesting any change be made, I leave that up to other reviewers to decide if this matters.

Reply:
That makes sense to me.  Of the `lodash` function replacements so far, this is the one I'm least confident about in terms of behaviour change, safety, and also readability to some extent.  I've only update this single callsite so far, but will re-consider the other locations too.

Reply:
(adding additional test coverage is probably the best answer to the safety and behaviour change concerns, and I should look into that too)